tuf/client/updater.py

```diff
@@ -228,6 +232,14 @@ def __init__(self, map_file):
     #  }
     self.repository_names_to_mirrors = self.map_file['repositories']
 
+    self.targets_map_filename = None
+
+    if targets_map_filename is not None:
+      # Is 'targets_map_file' a path?  If not, raise
+      # 'securesystemslib.exceptions.FormatError'.  The actual content of the map
+      # file is validated later on in this method.
```

Comment:
Please don't copy those comments. They are already everywhere (not your fault) and IMO not particularly helpful.